playbooks/roles/mongo_3_2/templates/backup-mongo.sh.j2

```diff
@@ -0,0 +1,67 @@
+#!/bin/bash
+# ref https://tasks.opencraft.com/browse/SE-1669
+# Script to perform a point-in-time dump of the local mongodb database using
+# mongodump.
+# includes locking (prevent this script from running multiple times in
+# parallel), creating a snapshot of the volume used to backup to, and snitch monitoring
+
+# exit by default on failure
+set -e
+
+# vars
+ARCHIVE_NAME="mongo-backup-$(date --iso-8601=minutes --utc).gz"
+VOLUME="{{ MONGO_BACKUP_EBS_VOLUME_DEVICE }}" # eg. /dev/svdk
+VOLUME_ID="{{ MONGO_BACKUP_EBS_VOLUME_ID }}" # eg. vol-123456
+VOLUME_MOUNT_PATH="{{ MONGO_BACKUP_VOLUME_MOUNT_PATH }}" # eg. vol-123456
+SNITCH_URL="{{ MONGO_BACKUP_SNITCH_URL }}"
```

Comment:
Cool -- all of these new variables are going to need defaults in the `roles/mongo_3_2/defaults/main.yml` file, and the possibility of `!!null` handled in this script.

Reply:
oh something just occurred to me - if `mongo_3_2` refers to this being a cluster of 3, then shouldn't we copy to a new role and name it mongo_4_X instead? (idk what the 2 refers to)

Reply:
The `3_2` in this case refers to mongo version 3.2, so we're ok :)

Reply:
Aaahh